types/caseData.d.ts

```diff
@@ -136,6 +136,10 @@ interface CaseData {
   witnessDetails?: Collection<WitnessDetails>[];
   interpreterLanguage?: Collection<InterpreterLanguage>[];
   isDecisionAllowed?: string;
+  updateTribunalDecisionList?: string;
+  typesOfUpdateTribunalDecision?: DynamicList;
```

Comment:
Do we need this field? think we can omit it.

Reply:
removed the case field "typesOfUpdateTribunalDecision" and "decisionAndReasonDocsUpload"